docs/vectorizer-api-reference.md

```diff
@@ -343,6 +344,49 @@ The function takes several parameters to customize the Ollama embedding configur
 
 A JSON configuration object that you can use in [ai.create_vectorizer](#create-vectorizers).
 
+### ai.embedding_voyageai
+
+You use the `ai.embedding_voyageai` function to use a Voyage AI model to generate embeddings.
+
+The purpose of `ai.embedding_voyageai` is to:
+- Define which Voyage AI model to use.
+- Specify the dimensionality of the embeddings.
+- Configure the model's truncation behaviour, and api key name.
+- Configure the input type.
+
+#### Example usage
+
+This function is used to create an embedding configuration object that is passed as an argument to [ai.create_vectorizer](#create-vectorizers):
+
+```sql
+SELECT ai.create_vectorizer(
+    'my_table'::regclass,
+    embedding => ai.embedding_voyageai(
+      'voyage-3-lite',
+      512,
+      truncate => false,
```

Comment:
I believe we should not set the `truncate` to `false` in all of our examples unless we explicitly want to show the behaviour when is set to `false`. Otherwise, we might confuse users, who 99.9% of the time will want this to be `true` as default.

```suggestion
```

Reply:
done